interface/main/mobile/m_functions.php

```diff
@@ -0,0 +1,800 @@
+            max-width:200px;}
+        
+        #documents_list {
+            width:100%;
+        }
+        .line_2_style {
+            min-height:17px;
+        }
+
+        body { margin: 0px 0px; padding: 0px; text-align:center; background-color:transparent !important;}
+        #first_name {
+            font-size: 12px;
+            float: right;
+        }
+        .space {
+            margin-bottom: 10px;
+            clear: both;
+        }
+
+        .padding_Info_text {
+            padding-top : 50px;
+            padding-bottom : 10px;
+        }
+
+        .arrow_right  {
+            position: relative;
+            border: 1px solid #c2e1f5;
+            border-radius: 10px;
+            display: block;
+            float: right;
+            clear:both;
+        }
+        .arrow_right:after, .arrow_right:before {
+            left: 100%;
+            top: 50%;
+            border: solid transparent;
+            content: " ";
+            height: 0;
+            width: 0;
+            position: absolute;
+            pointer-events: none;
+        }
+
+        .arrow_right:after {
+            border-color: rgba(30, 5, 252, 0);
+            border-left-color: #337ab7;;
+            border-width: 5px;
+            margin-top: -5px;
+        }
+
+        .arrow_left {
+            position: relative;
+            background: #E0E4E0;
+        }
+        .arrow_left:after, .arrow_left:before {
+            right: 99%;
+            top: -5px;
+            border: solid transparent;
+            content: " ";
+            height: 0;
+            width: 0;
+            position: absolute;
+            pointer-events: none;
+        }
+
+        .arrow_left:after {
+            border-color: rgba(136, 183, 213, 0);
+            border-right-color: #E0E4E0;
+            border-width: 5px;
+            margin-top: 13px;
+        }
+        .dialogIframe {
+            border:none !important;
+            background-color: transparent;
+        }
+        @keyframes blink {
+            from, to { opacity: 1 }
+            90% { opacity: 0.2 }
+        }
+
+        .shrinkToFit {
+             height: 90%;
+        }
+        
+        .pname {
+            position:relative;
+            margin: 0px auto;
+            font-size:0.9em;
+            top:3px;
+            font-weight: bold;
+        }
+
+    </style>
+    
+    <?php
+}
+    
+    /**
+     * Common header for mobile pages.
+     * This was build from Google's mobile page.
+     * If the goal is pure bootstrap, this would be the place to start it off.
+     */
+function common_header($display = '')
+{
+    ?>
+        <header>
+            <div id="menu_top">
+                <a <?php
+                if ($display =='photo') {
+                    echo ' class="active" ';
+                } ?>
+                        href="<?php echo $GLOBALS['webroot']; ?>/interface/main/mobile/camera.php?v=<?php echo mt_rand(); ?>"><?php echo xlt('Imaging'); ?></a>
```

Comment:
Rather than mt_rand, recommend using the following:
$GLOBALS['v_js_includes']
(note there is a very cool mechanism for developers to makes these all be random numbers during development; check out this script: https://github.com/openemr/openemr/blob/master/.env.example ; basically copy it to .env, set dev,, and then it magically works)